### ISSUE TITLE
Update exports style in goog.module-migrated files

### ISSUE THREAD

**Issue by cpcallen** [google/blockly#5133]
### Background

Earlier versions of the `goog.module` Module Conversion Guide in #5026 specified the following export style, with a single assignment to `exports`:

```JS
  /** @public */
  const publicThing = /* ... */;
  /** @private */
  const supposedlyPrivateThing = /* ... */;
  /** @private */
  const actuallyPrivateThing = /* ... */;

  exports = {
    publicThing,
    supposedlyPrivateThing_: supposedlyPrivateThing,
  };
```

After some further consideration, we decided it would be preferable to organise exports as a series of assignments to individual properties on `exports` instead (**final style TBC**):

```
const publicThing = /* ... */;
exports.publicThing = publicThing;

const supposedlyPrivateThing = /* ... */;
exports.supposedlyPrivateThing_ = supposedlyPrivateThing;

const actuallyPrivateThing = /* ... */;
```

### Issue

We should be consistent about this.

## Suggested Fix

Create a single PR bulk-updating already-migrated files from the earlier style to the current one.

**Comment by cpcallen** [comment]
We decided in the meeting on 2021-07-20 to also add `/* eslint-disable-next-line no-unused-vars */` to suppress warnings from eslint due to `goog.requireType` (and make the warning back into an error, as it was previously) at the same time.

**Comment by cpcallen** [comment]
This has mostly been fixed by #5148.  All that is left is to clean up any remaining eslint warning(s) (I think there is one in a not-yet-migrated file) and then reenable generating errors instead of warnings.

**Comment by cpcallen** [comment]
The lint errors were resolved quite some time ago, so I'm going to close this.
